msgs/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ install(FILES ${WEARABLEDATA_FILES}
 yarp_add_idl(WEARABLEACTUATORS_FILES thrift/WearableActuators.thrift)
 
 add_library(WearableActuators ${WEARABLEACTUATORS_FILES} thrift/WearableActuators.thrift)
-add_library(Wearable::WearableActuators ALIAS WearableActuators)
+add_library(WearableActuators::WearableActuators ALIAS WearableActuators)
```

Comment:
Did the alias change? This might be breaking in some private code (cc @dariosortino)

Reply:
Tagging @davidegorbani  as he is more familiar in writing cmake for ifeel applications

Reply:
The alias is only used for code internal to the `wearables`/`human-dynamics-estimation`, or project that included them via `FetchContent`/`add_subdirectory` (none, if I am not wrong), so we should not risk any regression.

The reason why I changed is that before this PR (and the merge PR), this was the situation in wearables:

| Target name during `build` | Imported target name after install |
|:---------------------------:|:--------------------------------:|
| `Wearable::IWear`         |  `IWear::IWear` |
|  `Wearable::WearableData` | `WearableData::WearableData` |
| `Wearable::WearableActuators` | `WearableActuators ::WearableActuators` |

The first one was already changed in the merge commit, while the last two just in this PR.

The whole idea of the ALIAS targets is to be able to link to the same target in the code that consumes the library when built in the same project and the code that consumes the library via `find_package`, so using two different namespaces was basically making it useless. 

I can advertise better this change and if you are worried of somebody consuming the project via `add_subdirectory` I could even add back the `Wearable::` alias and deprecating them, but I am not sure the gains are worth the work, even if it is minimal.

Reply:
Ah I see. Indeed, the math I found in the private code was using ``WearableActuators::WearableActuators``. No problem then!

Reply:
Just in case, I updated the PR messages in https://github.com/robotology/human-dynamics-estimation/pull/398 and https://github.com/robotology/human-dynamics-estimation/pull/395 to be explicit about this change.